package.json

```diff
@@ -5,7 +5,8 @@
   "description": "Fastish HTML",
   "main": "index.js",
   "engines": {
-    "node": "^4.0.0"
+    "node": "^4.7.0",
+    "npm": "^5"
```

Comment:
Nit: 5.0.0 to be consistent with [semantic versioning](http://semver.org/).

Reply:
Done